website/app/styles/components/font-helpers-list.scss

```diff
@@ -0,0 +1,33 @@
+// DOC FONT HELPERS LIST
+
+@use "../typography/mixins" as *;
+
+.doc-font-helpers-list {
+  margin: 1.5rem 0;
+  padding: 0;
+  list-style: none;
+}
+
+.doc-font-helpers-list__item {
+  display: flex;
+  align-items: center;
+  border: 1px solid var(--doc-color-gray-500);
+  border-radius: 3px;
+
+  & + & {
+    margin-top: 16px;
+  }
+}
+
+.doc-font-helpers-list__preview {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 120px;
+  height: 58px;
```

Comment:
Could we change the `height` to `62px` so the divider goes from top to bottom without gaps?

**58px**
<img width="687" alt="Screen Shot 2022-12-13 at 3 14 35 PM" src="https://user-images.githubusercontent.com/2142312/207465939-aff17916-f3bf-4f1f-9c0f-625e0d4cae42.png">

**62px**
<img width="698" alt="Screen Shot 2022-12-13 at 3 14 25 PM" src="https://user-images.githubusercontent.com/2142312/207465955-1fd688e1-7f76-49a8-9f06-404fbc4f2d1c.png">

Reply:
fixed by stretching it to fill the parent, instead of having a fixed height